CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - #885 - handle symlinks when using remote docker.
 - #868 - ignore the `CARGO` environment variable.
 - #867 - fixed parsing of `build.env.passthrough` config values.
+- #554 - Add `--force-non-host` flag.
```

Comment:
wrong entry, should be  a .changes file

Reply:
with the Github cli it's easy

`gh pr checkout 554`, it sets up the remote and branch (with correct upstream ref)

 
also possible to do with dev.github.com on the pr, just add `dev.*` to the url

Reply:
I've tried that, it keeps pushing to the wrong remote.

Reply:
Wait this worked for the first time ever.